portfolio/src/main/webapp/script.js

```diff
@@ -112,18 +112,53 @@ window.addEventListener('DOMContentLoaded', (event) => {
     showSlides(slideIndex);
 });
 
-function getMessage() {
-  fetch('/data').then(response => response.json()).then((comment) => {
-
-    const commentElement = document.getElementById('greeting');
-    commentElement.innerHTML = '';
-    commentElement.appendChild(
-        createListElement(comments[0]));
+function getComments() {
+  const limit = document.getElementById('input').value;
+  fetch('/data?limit-input=' + limit).then(response => response.json()).then((comments) => {
+    const commentElement = document.getElementById('comments-container');
+    commentElement.innerHTML = "";
+    comments.forEach((comment) => {
+      commentElement.appendChild(createCommentElement(comment));
+    })
   });
 }
 
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createCommentElement(comment) {
+  const commElement = document.createElement('p');
+  commElement.className = 'comment';
+
+  const mainElement = document.createElement('span');
+  mainElement.innerText = comment.name + ": " + comment.comment;
+
+  commElement.appendChild(mainElement);
+  return commElement;
+}
+
+function deleteComments() {
+  fetch('/delete-data', {method: 'POST'}).then(() => getComments());
+}
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Animal');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Lions', 10],
+          ['Tigers', 5],
+          ['Bears', 15]
+        ]);
+
+  const options = {
```

Comment:
Something we often do when adding placeholders is add a TODO. For example, `TODO(yourname): Update this with a data fetch`

Reply:
Gotcha!